.github/workflows/collector-slim.yml

```
@@ -33,7 +34,9 @@ jobs:
         if: github.event_name != 'pull_request'
         run: |
           echo 'BUILD_BUILDER_IMAGE=true' >> "$GITHUB_ENV"
+          echo 'BUILD_MULTIARCH_IMAGE=true' >> "$GITHUB_ENV"
           echo "COLLECTOR_BUILDER_TAG=${DEFAULT_BUILDER_TAG}" >> "$GITHUB_ENV"
```

There's a shellcheck lint error being reported here because there's more than 2 echo commands appending to the same file, could you rewrite it like this?
```suggestion
          {
            echo 'BUILD_BUILDER_IMAGE=true'
            echo 'BUILD_MULTIARCH_IMAGE=true'
            echo "COLLECTOR_BUILDER_TAG=${DEFAULT_BUILDER_TAG}"
          } >> "$GITHUB_ENV"
```

You can test this locally with [actionlint](https://github.com/rhysd/actionlint)